ui/src/app/components/object-explorer/object-explorer.component.html

```diff
@@ -1,5 +1,9 @@
 <div class="container">
-  <mat-tab-group [ngClass]="[isLeftColumnCollapse ? 'hidden' : 'display']" selectedIndex="1">
+  <mat-tab-group
```

Comment:
https://drive.google.com/file/d/1tkeikLv0GzkPcv5kGWjukmwwR__KTiPT/view?usp=sharing&resourcekey=0-FOm_fyVTWLHFjm5z4qUYxA: the object explorer is showing click on the object name on the Spanner draft panel instead of source database

Reply:
@shreyakhajanchi **"To view an object details, click on the object name on the [source database] panel."**

Reply:
Should we change it too something like this when we are in source panel and didn't select any object?

Reply:
yes